### PR TITLE
fix: fix addWriteEvent error

### PR DESCRIPTION
**Pull request by marsevilspirit**
error sush as:
image

This errno error code indicates that the write event already exists.
This error caused the single-core CPU to spike (because the flag bit was not successfully set, the write event was not successfully deleted).

Set the correct flag bit, and this problem will be solved.

## Summary by CodeRabbit

- **New Features**
	- Enhanced socket handling for improved network communication.
- **Bug Fixes**
	- Refined logic for checking socket readiness, ensuring more reliable packet sending.

### CODE REVIEW

**Review by AlexStocks — APPROVED** [review]

File: src/net/thread_manager.h [hunk 1]
```diff
@@ -255,7 +256,7 @@ void ThreadManager<T>::SendPacket(const T &conn, std::string &&msg) {
 
   connPtr->netEvent_->SendPacket(std::move(msg));
 
-  if (connPtr->netEvent_->CheckSetFlag(0)) {
+  if (connPtr->netEvent_->CheckSetFlag(static_cast<uint8_t>(IOSocketFlag::WRITE))) {
```
Comment (lqxhub) [inline src/net/thread_manager.h:259]: 这样修改虽然能解决CPU占用100% 的问题,但还是会出现 多线程 修改 epoll flag,导致的数据写入失败, 对于修改 epoll这部分的逻辑,可能要重新设计了.

Comment (lqxhub) [reply src/net/thread_manager.h:259]: > 这样修改虽然能解决CPU占用100% 的问题,但还是会出现 多线程 修改 epoll flag,导致的数据写入失败, 对于修改 epoll这部分的逻辑,可能要重新设计了.

之前因为这里写错了, 误打误撞,覆盖了多线程修改 epoll的问题

Comment (marsevilspirit) [reply src/net/thread_manager.h:259]: 如果需要重新设计的话，那我就关掉这个pr了

Comment (lqxhub) [reply src/net/thread_manager.h:259]: OK

**Comment by marsevilspirit** [comment]
Because this part needs to be redesigned, I am closing this PR.

**Pull request closed without merging**